pkg/cloudprovider/suite_test.go

```diff
@@ -770,6 +774,20 @@ var _ = Describe("CloudProvider", func() {
 				Expect(err).NotTo(HaveOccurred())
 				Expect(isDrifted).To(BeEmpty())
 			})
+			It("should not return drifted if karpenter.sh/ec2nodeclass-hash-version annotation version does not match the EC2NodeClass on the NodeClaim", func() {
+				nodeClass.ObjectMeta.Annotations = map[string]string{
+					v1beta1.AnnotationEC2NodeClassHash:        "test-hash-1",
```

Comment:
Same comment as the other PR: These values need to be different between the NodeClaim and the EC2NodeClass or you aren't really evaluating anything in your testing

Reply:
This still doesn't look like it was fixed. The hash values for both resources are still the same.

Reply:
It was left out due rebase issue. Should be updated now